src/calibrate-multi-hop/requirements.txt

```diff
@@ -0,0 +1,8 @@
+pandas
+numpy
+pymongo>=3.11.3
+Flask_Cors>=3.0.10
+Flask>=1.1.2
+Flask_PyMongo>=2.3.0
+python-dotenv>=0.17.1
+git+https://github.com/airqo-platform/AirQo-modules.git@calibration-simple#egg=simple_calibration&subdirectory=calibration/simple_calibration
```

Comment:
Hi @bobturneruk and @Priscilla-AL 
Lovely, so looks like this is where the package is consumed from.
Could we explore publishing these packages? How is the process like? Smooth or complicated for our usecase?
A little search made me land on this: https://www.codementor.io/@ajayagrawal295/how-to-publish-your-own-python-package-12tbhi20tf

Reply:
It's fine - I'm very familiar with it. It should work well.

Reply:
Awesome, that will be great! Looking forward to trying it out. Perhaps a short workshop of sorts? @bobturneruk